ipatests/test_integration/test_acme.py

```diff
@@ -114,6 +115,11 @@ def test_enable_acme_service(self):
         else:
             raise exc
 
+    def test_centralize_acme_enable(self):
+        """Test if ACME enable on replica if enabled on master"""
+        result = self.master.run_command(['ipa-acme-manage', 'status'])
+        assert 'enabled' in result.stdout_text
```

Comment:
Seems like it is checking the replica now.

I think we may want some retry behavoiur here in case there is a lag in the replica receiving and responding to the new configuration.  (Same for the disable check).  What do you think @mrizwan93 @rcritten ?